calculator.js

```diff
@@ -0,0 +1,83 @@
+//array of numbers for our buttons
+var array = [1, 2, 3, 4, 5, 6, 7, 8, 9, "+", "-", "*"]
+
+//makes the Input
+
+function makeInput() {
+    var mkInput = document.createElement("input");
+    mkInput.className = "cal-input";
+    mkInput.id = "DtaInput";
```

Comment:
I strongly recommend keeping `id` attributes all lower case, dash separated if necessary.

That being said, you don't even need an `id` attribute. You could grab the calculator input by calling `document.querySelector('.cal-input')`.